test/fs/test_write.cpp

```diff
@@ -24,7 +24,7 @@ int main()
         var pos = lengthBytesUTF8(str);
 
         str = '1234567890123456789012345678901234567890123456789012345678901234567890123456789012345678901234567890';
-        data = new Uint8Array(100);
+        data = new Uint8Array(101);
```

Comment:
Can you do `str.length` there instead maybe?

Reply:
1da12e4f7acf8a885ce199d98b136cbc3002e823